content/references/rippled-api/transaction-formats/transaction-types/accountset.md

```diff
@@ -42,7 +42,11 @@ The `Domain` field is represented as the hex string of the lowercase ASCII of th
 
 To remove the `Domain` field from an account, send an AccountSet with the Domain set to an empty string.
 
-Client applications can use the [ripple.txt](https://wiki.ripple.com/Ripple.txt) file hosted by the domain to confirm that the account is actually operated by that domain.
+You can put any domain in your account's `Domain` field. To prove that an account truly belongs to the same person or business as a domain, Ripple recommends establishing a "two-way link":
```

Comment:
Does anything truly belong to anyone? /s Perhaps that second sentence could be made stronger, also, adding a suggestion to improve clarity:
> To prove that an account and domain belong to the same person or business, Ripple recommends establishing a "two-way link":